### PR TITLE
Update README to reflect IPV in the heading

### PR DESCRIPTION
**Pull request by galund**
- we could consider changing the name of the repository, but I don't think that's particularly important

## Why

Because just referring to Authentication is misleading.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
-# Authentication technical documentation
+# GOV.UK Sign In technical documentation
 
-TBA
+This is the documentation for relying parties that want to connect their service to the GOV.UK Sign In product - for authentication, and optionally for identity proofing and verification.
```
Comment (elenaschwan) [inline README.md:3]:
```suggestion
This documentation is for government services that want to integrate with GOV.UK One Login to 

* authenticate their users
* verify their user's identity  
```

**Review by elenaschwan — COMMENTED** [review]
@galund I just stumbled over this - shall we get the README file updated after all this time?

**Comment by huwd** [comment]
Given the age of this, @galund, @sabrinaharris, @ImogenCraigmile, @elenaschwan, are you happy for me to close this down.

And we re-address the topic of naming the repo in a more holistic way elsewhere?

**Comment by huwd** [comment]
I've rebased against main, and fixed up @elenaschwan's changes.

Now just needs a content review and ✅ from @sabrinaharris or @ImogenCraigmile

File: README.md [hunk 2]
```diff
@@ -1,6 +1,9 @@
-# Authentication technical documentation
+# GOV.UK One Login technical documentation
 
-TBA
+This documentation is for government services that want to integrate with GOV.UK One Login to 
+
+* authenticate their users
+* verify their user's identity
```
Comment (galund) [inline README.md:6]:
```suggestion
* verify their users' identity  
```

Looks like I dug myself in a grammar hole here :-o

**Comment by huwd** [comment]
Cool cool, @galund will fix that up for commit history cleanliness

**Comment by galund** [comment]
Hah I have never understood the GDS obsession with commit history cleanliness (I do like a straight-line history but removing the history of our spelling mistakes seems like a weird pretence at personal perfection). But yes I can rebase.

**Comment by huwd** [comment]
@galund I intend to give a talk to the tech community on just that theme in the next few weeks.
I think git history really matters, it's a really rich vein of documentation and safe rollback points if we do it well.

**Comment by huwd** [comment]
Bit annoying I've locked you out of being able to push @galund 🤔.
I think it's time to broaden these access controls out again. Apologies for the premature clamp down.

I've fixed up your commit. So @ImogenCraigmile just with you for the content review I think?

**Comment by ImogenCraigmile** [comment]
Just looking at it now!

**Review by ImogenCraigmile — APPROVED** [review]

**Review by huwd — APPROVED** [review]

**Pull request merged**